benchmark/benchmark_runner.cpp

```diff
@@ -385,6 +397,14 @@ int main(int argc, char **argv) {
 	LoadInterpretedBenchmarks(*fs);
 	parse_arguments(argc, argv);
 	const auto configuration_error = run_benchmarks();
+	
+	if (!summary.empty() && summarize) {
+		std::cout << "\n===============================  FAILURES SUMMARY  ===============================\n" << std::endl;
+		for (size_t i = 0; i < summary.size(); i++) {
+			std::cout << i + 1 << ". " << summary[i] << std::endl;
```

Comment:
```suggestion
			std::cout << summary[i] << std::endl;
```
I would probably remove the counter, and possibly add some form of line / spacing between reports.
Or add the counters, but as part of an header.
So that it's clear where they end.